core/src/main/kotlin/org/evomaster/core/output/TestSuiteSplitter.kt

```diff
@@ -107,16 +107,27 @@ object TestSuiteSplitter {
 
     private fun execSummary(clusters : MutableMap<String, MutableList<MutableList<RestCallResult>>>,
                             solution: Solution<RestIndividual>,
+                            oracles: PartialOracles,
                             splitResult: SplitResult
             ) : Solution<RestIndividual> {
-        val execSol = mutableListOf<EvaluatedIndividual<RestIndividual>>()
+        val execSol = mutableSetOf<EvaluatedIndividual<RestIndividual>>()
```

Comment:
a set requires immutability of its element. in general u cannot have a set of objects that can be mutated. u can use a temporary set to enforce uniqueness (which is fine) before converting to a list (as done here), but in that case you should add some comment